aw_analysis/query2_functions.py

```diff
@@ -126,13 +126,14 @@ def q2_query_bucket_eventcount(datastore: Datastore, namespace: TNamespace, buck
 
 
 @q2_function
-def q2_filter_keyvals(events: list, key: str, vals: list) -> List[Event]:
+def q2_filter_keyvals(events: list, key: str, vals: list, exclude: bool = False) -> List[Event]:
```

Comment:
I'm not a fan of this.
I think it's a clearer API if we have two functions with descriptive names instead of one function with more advanced options to be able to accomplish two things.

Reply:
Fair enough.